tests/api_workflow/test_api_workflow_compute_worker.py

```diff
@@ -114,4 +123,85 @@ def test_selection_config(self):
         )
         config = DockerWorkerConfig(worker_type=DockerWorkerType.FULL, selection=selection_config)
 
-        config_api = self._check_if_openapi_generated_obj_is_valid(config)
\ No newline at end of file
+        config_api = self._check_if_openapi_generated_obj_is_valid(config)
+
+def test___selection_config_from_dict() -> None:
```

Comment:
Minor: I would suggest to ignore leading underscores when naming tests. But in any case, Three underscores are too excessive 🙂

Reply:
Ah yes, my bad :) I actually like double underscore for private functions because it avoids potential naming conflicts if we have `fun` and `_fun` in the same file.

Reply:
You should actually never do that anyway:
1) to avoid confusion
2) to easily switch private/public indication

(I know I did it in a previous PR, I should not have done it :))

But I'm fine with the double underscore convention too.